code/obj/item/tank.dm

```diff
@@ -316,6 +316,18 @@ Contains:
 	name = "gas tank"
 	icon_state = "empty"
 
+/obj/item/tank/filled_custom
+	name = "custom gas tank"
+	icon_state = "empty"
+	var/custom_gas_type = "oxygen" // replace with exact varname of choice
+	var/custom_temperature = 175 KELVIN
+
+	New()
+		..()
+		src.air_contents.temperature = src.custom_temperature
+		src.air_contents.vars[src.custom_gas_type] = (10 * ONE_ATMOSPHERE) * src.air_contents.volume / (R_IDEAL_GAS_EQUATION * src.air_contents.temperature)
+
+
```

Comment:
What dis for?

Reply:
woops forgot to remove that. custom tank was used for testing with atlas so I could just throw a maxcap mix in and not varedit it 20 times. dont know how i missed it, thank u